pkgs/applications/science/molecular-dynamics/gromacs/default.nix

```diff
@@ -1,11 +1,13 @@
 { stdenv
 , fetchurl
 , cmake
-, singlePrec ? true
-, mpiEnabled ? false
+, hwloc
 , fftw
 , openmpi
 , perl
+, singlePrec ? true
+, mpiEnabled ? false
+, cpuAcceleration ? "SSE4.1"
```

Comment:
Do you know which possibilities this string can have? If only one is possible maybe it should be a bool.

Reply:
It let's you choose the level SIMD acceleration. According to the cmake file the following values are possible:
```
AUTO None SSE2 SSE4.1 AVX_128_FMA AVX_256 AVX2_256 AVX2_128 AVX_512 AVX_512_KNL MIC ARM_NEON ARM_NEON_ASIMD IBM_VMX IBM_VSX Sparc64_HPC_ACE Reference
```

Reply:
I am not sure if it is worth it adding a comment of the possible values. Probably not.